python/lsst/ctrl/mpexec/dotTools.py

```diff
@@ -233,10 +234,20 @@ def expand_dimensions(dimensions):
     allDatasets = set()
     if isinstance(pipeline, Pipeline):
         pipeline = pipeline.toExpandedPipeline()
+
+    # The next two lines are a workaround until DM-29658 at which time metadata
+    # connections should star working with the above code
```

Comment:
star -> start? 

Tim reminded me on other PR that comments need to be full sentences, i.e. have a period at the end.